### PR TITLE
Круглов - Лабораторная работа #1

### PR DESCRIPTION
**Pull request by EOKruglov**

### CODE REVIEW

**Comment by Kasmazyuk** [comment]
:+1:

File: modules/complex-number/test/test_kruglov_yegor_complex_number.cpp [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    ComplexNumber z3(2.0, 0.0);
+
+    // Act
+    z1 = z1 / z2;
+
+    // Assert
+    EXPECT_EQ(z3, z1);
+}
+
+TEST(Kruglov_Yegor_ComplexNumberTest, Can_Not_Divide_By_Zero) {
```
Comment (alexbatashev) [inline modules/complex-number/test/test_kruglov_yegor_complex_number.cpp]: Duplicate

File: modules/complex-number/test/test_kruglov_yegor_complex_number.cpp [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+    // Act
+    z1 = z1 / z3;
+
+    // Assert
+    EXPECT_EQ(z2, z1);
+}
+
+TEST(Kruglov_Yegor_ComplexNumberTest, Can_Divide_By_Complex_Num) {
+    // Arrange
```
Comment (alexbatashev) [inline modules/complex-number/test/test_kruglov_yegor_complex_number.cpp]: Duplicate

File: modules/complex-number/test/test_kruglov_yegor_complex_number.cpp [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+    ComplexNumber z1;
+
+    // Act
+    z1 = z;
+
+    // Assert
+    EXPECT_EQ(z, z1);
+}
+
+TEST(Kruglov_Yegor_ComplexNumberTest, Can_Add_Two_Numbers) {
```
Comment (alexbatashev) [inline modules/complex-number/test/test_kruglov_yegor_complex_number.cpp]: Duplicate

File: modules/complex-number/test/test_kruglov_yegor_complex_number.cpp [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+    ComplexNumber z2(20.0, 20.0);
+
+    // Act
+    z1 = z1 + z1;
+
+    // Assert
+    EXPECT_EQ(z2, z1);
+}
+
+TEST(Kruglov_Yegor_ComplexNumberTest, Can_Subtract_One_From_The_Other) {
```
Comment (alexbatashev) [inline modules/complex-number/test/test_kruglov_yegor_complex_number.cpp]: Duplicate

**Comment by EOKruglov** [comment]
@VadimKutovoi

**Comment by EOKruglov** [comment]
@alexbatashev corrected this

**Pull request merged**